metadata.json

```diff
@@ -1,8 +1,7 @@
 {
     "shell-version": [
         "3.38",
-        "40.beta",
-        "40.rc"
+        "40.0"
```

Comment:
Maybe just use `40` and squash the commit? That's what's done for the officially supported ones at https://gitlab.gnome.org/GNOME/gnome-shell-extensions/

Reply:
FTR, I'd recommend using really just `40` instead of `40.0`

Reply:
To explain: from 40 on the major version signals API stability - independent of the point release or beta/release candidate. And API stability is everything we are interested in as extension.

Reply:
Uploading new version, right now.